diff --git a/src/app/page.tsx b/src/app/page.tsx
index ced131b..5c21a8e 100644
--- a/src/app/page.tsx
+++ b/src/app/page.tsx
@@ -32,20 +32,9 @@ const Page = async () => {
       <Content>
         <h2>{t('about.name')}</h2>
 
-        <Suspense>
-          <Stats />
-        </Suspense>
-
-        <P><Trans i18nKey="about.content.p1" t={t} i18n={i18n}>_<br /><Link href="/how-to" rel="help">_</Link>_</Trans></P>
-
         <Video />
 
         <DownloadButtons />
-
-        <P><Trans i18nKey="about.content.p3" t={t} i18n={i18n}>_<a href="https://bengrant.dev" target="_blank" rel="noreferrer noopener author">_</a>_</Trans></P>
-        <P><Trans i18nKey="about.content.p4" t={t} i18n={i18n}>_<a href="https://github.com/GRA0007/crab.fit" target="_blank" rel="noreferrer noopener">_</a>_<Link href="/privacy" rel="license">_</Link>_</Trans></P>
-        <P>{t('about.content.p6')}</P>
-        <P>{t('about.content.p5')}</P>
       </Content>
     </Section>
 
diff --git a/frontend/src/components/Video/Video.tsx b/frontend/src/components/Video/Video.tsx
index 61e782d..ae82db8 100644
--- a/src/components/Video/Video.tsx
+++ b/src/components/Video/Video.tsx
@@ -16,7 +16,7 @@ const Video = () => {
       <iframe
         width="560"
         height="315"
-        src="https://www.youtube.com/embed/yXGd4VXZzcY?modestbranding=1&rel=0&autoplay=1"
+        src="https://www.youtube.com/embed/LDU_Txk06tM?modestbranding=1&rel=0&autoplay=1"
         title={t('video.title')}
         frameBorder="0"
         allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
@@ -26,7 +26,7 @@ const Video = () => {
   ) : (
     <a
       className={styles.preview}
-      href="https://www.youtube.com/watch?v=yXGd4VXZzcY"
+      href="https://www.youtube.com/watch?v=LDU_Txk06tM"
       target="_blank"
       rel="nofollow noreferrer"
       onClick={e => {
